src/app/components/radio/index.html

```diff
@@ -1,13 +1,267 @@
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-1.png" />
+                    <div class="caption caption-success sky-text-success">Do use radio buttons for single selections when the number of choices is low.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use radio buttons if the user’s decision will make a small number of other inputs relevant or irrelevant. The conditional inputs are visible and disabled when their activating radio is not selected.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-2.png" />
+                    <div class="caption caption-success sky-text-success">Do let radio buttons conditionally enable more inputs as part of a task.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use an icon radio group when choices can be clearly communicated with icons, and can be combined under a single field or question. </p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-3.png" />
+                    <div class="caption caption-success sky-text-success">Do use icon radio groups when choices can be clearly illustrated with icons.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use individual icon radio buttons when the radio group is spread across multiple rows or objects.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-4.png" />
+                    <div class="caption caption-success sky-text-success">Do use icon radio buttons when the choices cannot be grouped together.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <h3 class="sky-text-danger">Don't use when</h3>
```

Comment:
All of the Do/Don't visual examples are stacked instead of side-by-side when they would fit. Is that intentional?

Reply:
No. 😡Need to fiddle with it some more.

Reply:
Resolving since the visual examples no longer seem to be stacked.